common-util/functions/proposal.ts

```diff
@@ -0,0 +1,54 @@
+import { ethers } from 'ethers';
+import { isNil } from 'lodash';
+
+import { ethersToWei, formatToEth } from 'common-util/functions';
+import { ModuleDetails } from 'store/types';
+import { VEOLAS_QUORUM } from 'util/constants';
+
+// 2 million veolas in wei
+const quorumInWei = ethersToWei(`${VEOLAS_QUORUM}`);
+
+/**
+ * check if the current proposal has enough veOLAS to be executed
+ */
+export const getCurrentProposalInfo = (
+  proposal: ModuleDetails['scheduled_tweet']['tweets'][number],
+) => {
+  // example of voters: [ { '0x123': '1000000000000000000000000' } ]
+  const totalVeolasInWei = proposal?.voters?.reduce((acc, voter) => {
+    // previously the voters were stored as an [account]: balance.
+    // now, it is stored as an object (eg. Check "Voter" in prop-types.js).
+    const currentVeOlasInWei = !isNil(voter?.votingPower)
+      ? ethersToWei(`${voter?.votingPower || '0'}`)
+      : Object.values(voter)[0];
```

Comment:
can be straightforward.
```suggestion
    const currentVeOlasInWei = isNil(voter?.votingPower)
      ? Object.values(voter)[0];
      : ethersToWei(`${voter?.votingPower || '0'}`)
```

Reply:
i realised `Object.values(voter)[0];` this isn't needed.
all the voters in the new API follow the same contract, eg:
```js
  /**
   * voter: {
   *  address: '0x123',
   *  signature: '0x789',
   *  votingPower: '1000000000000000000000000',
   * }
   */
```